client/packages/coldchain/src/Equipment/api/hooks/log/useAssetLogInsert.ts

```diff
@@ -0,0 +1,22 @@
+import {
+  InsertAssetLogInput,
+  useMutation,
+  useQueryClient,
+} from '@openmsupply-client/common';
+import { useAssetApi } from '../utils/useAssetApi';
+
+export const useAssetLogInsert = () => {
+  const queryClient = useQueryClient();
+  const api = useAssetApi();
+
+  return useMutation(
+    async (log: Partial<InsertAssetLogInput>) => api.insertLog(log),
+    {
+      onSettled: assetId =>
+        queryClient.invalidateQueries(api.keys.logs(assetId ?? '')),
```

Comment:
Do we need to invalidate the assets as well? As they have statuses from a loader which might be incorrect now?

Reply:
good point! have changed to use the base key and invalidated. now working